packages/components/src/TextField/TextField.tsx

```diff
@@ -79,6 +86,19 @@ export const TextField = forwardRef<HTMLDivElement, TextFieldProps>(
       return <></>;
     }, [helperTextProp, success, error]);
 
+    const handleOnBlur = (event: FocusEvent<HTMLInputElement>) => {
```

Comment:
```suggestion
    const handleBlur = (event: FocusEvent<HTMLInputElement>) => {
```

https://kaluga-astral.github.io/style-guide/docs/rules/react/logic#%D0%BE%D0%B1%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D1%87%D0%B8%D0%BA%D0%B8-%D1%81-%D0%BF%D1%80%D0%B5%D1%84%D0%B8%D0%BA%D1%81%D0%BE%D0%BC-handle